src/browser/modules/Stream/CypherFrame/RelatableView/relatable/global.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const content: any
+  export default content
```

Comment:
I think we already have this type in the our file called `custom.d.ts` so it's not needed here